tests/charts/test_flower.py

```diff
@@ -280,6 +280,25 @@ def test_should_add_extraEnvs(self):
             "spec.template.spec.containers[0].env", docs[0]
         )
 
+     def test_should_add_component_specific_labels(self):
+        docs = render_chart(
+            values={
+                "flower": {
+                    "enabled": True,
+                    "labels": {"test_label": "test_label_value"},
+                },
+            },
+            show_only=["templates/flower/flower-deployment.yaml"],
```

Comment:
You need to check all of the flower templates.

Reply:
done. Added for the rest of the pods as well